__tests__/app.test.js

```diff
@@ -43,19 +45,42 @@ describe.only("GET /api/topics",()=>{
     })
   })
 
-  test("200: Responds with an empty array when ",()=>{
+})
+
+describe("/api/articles/:article_id",()=>{
+  test("200: Responds with an article object, which should have the following properties:author,title,article_id,body,topic,created_at,votes and article_img_url",()=>{
     return request(app)
-    .get("/api/topics")
-    .then(({body:{topics}})=>{
-      expect(topics.length).toBe(3)
-      topics.forEach(topic=>{
-        expect(topic).toMatchObject({
-          slug:expect.any(String),
-          description:expect.any(String)
-        })
+    .get("/api/articles/2")
+    .expect(200)
+    .then(({body: {article}})=>{
+      expect(article).toMatchObject({
+        author:expect.any(String),
+        title:expect.any(String),
+        article_id:expect.any(Number),
```

Comment:
We should know what the article_id is at the very least! It should be 2 right? 😄 I would be more specific